qa/rolling-upgrade/src/test/java/org/opensearch/neuralsearch/bwc/rolling/RestNeuralStatsActionIT.java

```diff
@@ -22,78 +22,101 @@ public class RestNeuralStatsActionIT extends AbstractRollingUpgradeTestCase {
-                    getIndexNameForTest(),
-                    Files.readString(Path.of(classLoader.getResource("processor/IndexMappings.json").toURI())),
-                    PIPELINE_NAME
-                );
-                addDocument(getIndexNameForTest(), "0", TEST_FIELD, TEXT, null, null);
-
-                responseBody = executeNeuralStatRequest(new ArrayList<>(), new ArrayList<>());
-
-                assertEquals(numberOfExecution + 1, getNestedValue(parseAggregatedNodeStatsResponse(responseBody), EventStatName.TEXT_EMBEDDING_PROCESSOR_EXECUTIONS));
-                assertEquals(numberOfProcessor + 1, getNestedValue(parseInfoStatsResponse(responseBody), InfoStatName.TEXT_EMBEDDING_PROCESSORS));
-                break;
-            case MIXED:
-                modelId = getModelId(getIngestionPipeline(PIPELINE_NAME), TEXT_EMBEDDING_PROCESSOR);
-                loadModel(modelId);
-                addDocument(getIndexNameForTest(), "1", TEST_FIELD, TEXT, null, null);
-                addDocument(getIndexNameForTest(), "2", TEST_FIELD, TEXT, null, null);
-                addDocument(getIndexNameForTest(), "3", TEST_FIELD, TEXT, null, null);
-
-                // Get stats
-                responseBody = executeNeuralStatRequest(new ArrayList<>(), new ArrayList<>());
-
-                assertEquals(numberOfExecution + 3, getNestedValue(parseAggregatedNodeStatsResponse(responseBody), EventStatName.TEXT_EMBEDDING_PROCESSOR_EXECUTIONS));
-                assertEquals(numberOfProcessor, getNestedValue(parseInfoStatsResponse(responseBody), InfoStatName.TEXT_EMBEDDING_PROCESSORS));
-                break;
-            case UPGRADED:
-                try {
-                    modelId = getModelId(getIngestionPipeline(PIPELINE_NAME), TEXT_EMBEDDING_PROCESSOR);
-                    loadModel(modelId);
-                    addDocument(getIndexNameForTest(), "4", TEST_FIELD, TEXT, null, null);
-                    addDocument(getIndexNameForTest(), "5", TEST_FIELD, TEXT, null, null);
-                    addDocument(getIndexNameForTest(), "6", TEST_FIELD, TEXT, null, null);
-
-                    // Get stats
-                    responseBody = executeNeuralStatRequest(new ArrayList<>(), new ArrayList<>());
-
-                    assertEquals(
-                        numberOfExecution + 3,
-                        getNestedValue(parseAggregatedNodeStatsResponse(responseBody), EventStatName.TEXT_EMBEDDING_PROCESSOR_EXECUTIONS)
-                    );
-                    assertEquals(numberOfProcessor, getNestedValue(parseInfoStatsResponse(responseBody), InfoStatName.TEXT_EMBEDDING_PROCESSORS));
-                } finally {
-                    wipeOfTestResources(getIndexNameForTest(), PIPELINE_NAME, modelId, null);
-                }
-                break;
-            default:
-                throw new IllegalStateException("Unexpected value: " + getClusterType());
-        }
-    }
+//    public void testStats_E2EFlow() throws Exception {
```

Comment:
How about using @Disabled annotation instead of comment out? Example: https://github.com/opensearch-project/common-utils/blob/3e28c9218fb1229a49194a5850902b85697d770f/src/test/java/org/opensearch/commons/rest/IntegrationTests.java#L31

Reply:
It won't work as we are not using jupitor test framework.